### PR TITLE
commander-genius: Update Commander Genius to 2.2.2

### PR DESCRIPTION
**Pull request by RobLoach**
Update to 2.2.2...
https://github.com/gerstrong/Commander-Genius/releases/tag/v2.2.2

### CODE REVIEW

File: games/commander-genius/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/commander-genius*
```
Comment (strycore) [inline games/commander-genius/.gitignore]: Ususally, I don't put stiff in gitignore, this lets me see easily what was added by build scripts and clean everything up with git clean -df

**Comment by strycore** [comment]
```
/home/ubuntu/buildbot/games/commander-genius/commander-genius-src/GsKit/base/GsPython.cpp: In function ‘bool loadStrFunction(PyObject*, const string&, std::__cxx11::string&, int)’:
/home/ubuntu/buildbot/games/commander-genius/commander-genius-src/GsKit/base/GsPython.cpp:204:63: error: ‘PyUnicode_AsUTF8’ was not declared in this scope
             char *str = PyUnicode_AsUTF8(objectsRepresentation) ;
                                                               ^
/home/ubuntu/buildbot/games/commander-genius/commander-genius-src/GsKit/base/GsPython.cpp: In function ‘bool loadStrFunction(PyObject*, const string&, std::__cxx11::string&)’:
/home/ubuntu/buildbot/games/commander-genius/commander-genius-src/GsKit/base/GsPython.cpp:268:63: error: ‘PyUnicode_AsUTF8’ was not declared in this scope
             char *str = PyUnicode_AsUTF8(objectsRepresentation) ;
```

**Comment by RobLoach** [comment]
Strange, it builds for me just fine. What version of Cpp you on?

**Comment by strycore** [comment]
It's some dependency missing from the script

**Pull request merged**